client/components/teams/index.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="container container-top">
+    <div class="center">
+      <table class="table">
+        <thead>
+          <tr>
+            <th>Team name</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr
+            v-for="team in teams"
+            :key="team.id"
+            @click="viewTeam(team.id)">
+            <td>{{ team.name }}</td>
```

Comment:
If you only have one column here you don't actually need a table. You can just use a list